beacon-chain/blockchain/log.go

```diff
@@ -34,13 +35,22 @@ func logStateTransitionData(b block.BeaconBlock) {
 	if len(b.Body().VoluntaryExits()) > 0 {
 		log = log.WithField("voluntaryExits", len(b.Body().VoluntaryExits()))
 	}
-	if b.Version() == version.Altair {
+	if b.Version() == version.Altair || b.Version() == version.Bellatrix {
 		agg, err := b.Body().SyncAggregate()
 		if err == nil {
 			log = log.WithField("syncBitsCount", agg.SyncCommitteeBits.Count())
 		}
```

Comment:
This is a concerning unhandled error. 

Could you please fix while you are here?

Reply:
Nice catch. Fixed